honeybee_energy/config.py

```diff
@@ -0,0 +1,322 @@
+        return self._default_simulation_folder
+    
+    @default_simulation_folder.setter
+    def default_simulation_folder(self, path):
+        if not path:  # check the default location for simulations
+            path = self._find_default_simulation_folder()
+
+        self._default_simulation_folder = path
+
+        if not self.mute and self._default_simulation_folder:
+            print('Path to the default simulation folder is set to: '
+                  '{}'.format(self._template_library_folder))
+
+    @property
+    def energy_model_measure_path(self):
+        """Get or set the path to the energy_model_measure translating to OpenStudio.
+        
+        This folder must have the following sub-folders in order to be valid:
+            * ladybug - ruby gem with modules for model translation to OpenStudio.
+            * measures - folder with the actual measures that run the translation.
+        """
+        return self._energy_model_measure_path
+    
+    @energy_model_measure_path.setter
+    def energy_model_measure_path(self, path):
+        if not path:  # check the default locations of the energy_model_measure
+            path = self._find_energy_model_measure_path()
+
+        self._energy_model_measure_path = path
+        
+        # check that the library's sub-folders exist
+        if self._energy_model_measure_path:
+            assert os.path.isdir(os.path.join(path, 'ladybug')), \
+                '{} lacks a "ladybug" folder for the translation ruby gem.'.format(path)
+            assert os.path.isdir(os.path.join(path, 'measures')), \
+                '{} lacks a "measures" folder.'.format(path)
+            if not self.mute:
+                print('Path to the energy_model_measure is set to: '
+                      '{}'.format(self._energy_model_measure_path))
+
+    @property
+    def python_package_path(self):
+        """Get the path to where this Python package is installed."""
+        return os.path.split(os.path.dirname(__file__))[0]
+
+    @property
+    def python_exe_path(self):
+        """Get the path to the Python.exe file."""
+        return sys.executable
+
+    def load_from_file(self, file_path):
+        """Set all of the the properties of this object from a config JSON file.
+        
+        Args:
+            file_path: Path to a JSON file containing the file paths. A sample of this
+                JSON is the config.json file within this library.
+        """
+        # check the default file path
+        assert os.path.isfile(str(file_path)), \
+            ValueError('No such a file as {}'.format(file_path))
+
+        # set the default paths to be all blank / defauktes
+        default_path = {
+            "energyplus_path": r'',
+            "openstudio_path": r'',
+            "template_library_folder": r'',
+            "default_simulation_folder": r'',
+            "energy_model_measure_path": r''
+        }
+
+        with open(file_path, 'rb') as cfg:
```

Comment:
`rb` can cause issues in Python 2 versus 3.

```suggestion
        with open(file_path, 'r') as cfg:
```

Reply:
Ok.